tools/crawler/crawler.ts

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-process-exit */
```

Comment:
Since this is for testing, maybe have the `crawler` under [`tests/helpers/`](https://github.com/sonarwhal/sonar/tree/5c3eef7e8da391d6a363e19d507d41c44c8cd149/tests/helpers)?

Reply:
It is mainly for testing, but maybe someone will use it for something else. We've had teams inside Microsoft asking for similar things and there is #383 that is somehow related. I was planning on putting those things in there although maybe it's better to create a separate repo for that?

Reply:
> maybe it's better to create a separate repo for that?

Yes!

Reply:
Name of the repo?

Reply:
> Name of the repo?

Nothing interesting comes to mind, maybe `sonar-testing-ground`?

Reply:
OK, I'll create a new repo when I finish the current refactor 😊